src/bases.jl

```diff
@@ -817,13 +839,16 @@ function get_vector_cached!(M::AbstractManifold, Y, p, X, B::CachedBasis)
 end
 
 """
-    get_vectors(M::AbstractManifold, p, B::AbstractBasis)
+    get_vectors(M::AbstractManifold, p, B::AbstractBasis=get_basis(M, p, DefaultOrthonormalBasis()))
```

Comment:
Why do the others just need `DefaultOrthonormalBasis()` but this here the same wrapped in another call? And the code seems to do something different, namely not call `get_basis`?

Reply:
`get_vectors` is generally supposed to work with cached bases, so we need to get a cached basis first.

Reply:
Then we could maybe mention that in line 844 in the docs here?

Reply:
Sure, I've added a note there.